core/vm/interpreter.go

```diff
@@ -193,9 +193,6 @@ func (in *EVMInterpreter) Run(contract *Contract, input []byte, readOnly bool) (
 		// enough stack items available to perform the operation.
 		op = contract.GetOp(pc)
 		operation := in.cfg.JumpTable[op]
-		if operation == nil {
```

Comment:
Sorry to be constantly nagging about tracing haha but this is gonna do a lot of `CaptureState(opUndefined...)`

Reply:
`opUndefined` still aborts with `ErrInvalidOpCode` so there shouldn't really be a change there?

Reply:
> 

How this is a problem?

Reply:
@s1na Well, the fix is it's basically the same as for the previous one: whenever there are two outputs for the same step of execution (same pc twice in a row), then ignore the first and only care about the second one.

Reply:
> opUndefined still aborts with ErrInvalidOpCode so there shouldn't really be a change there?

@axic the change is that with this PR, it will do a capturestate prior to `execute`, which it didn't do previously -- erroring before it reached `execute`.

Reply:
`Gonna do a lot of...` was an exaggeration, since there aren't many txes which actually execute undefined opcodes, I think? but yeah not saying this is a show-stopper or anything. Mainly wanted to document the change.